content/tutorials/aws-persistence.textile

```diff
@@ -188,45 +190,57 @@ Now we have an app we can set up a webhook to AWS Lambda to trigger when a new m
+    <h1>Ably Webhook Lambda Test</h1>
+    <button id="publish" type="button" value="Connecting...">Click to Publish</button>
     <div id="result" style="padding: 10px"></div>
   </body>
 
-  <script type="text/javascript" src="https://cdn.ably.io/lib/ably.min-1.js"></script>  <!-- Include the latest Ably Library  -->
-  <script src="http://code.jquery.com/jquery-1.8.3.min.js"></script> <!-- Include jquery to automatically run javascript-->
+  <script
+    type="text/javascript"
+    src="https://cdn.ably.io/lib/ably.min-1.js"
+  ></script>
+  <!-- Include the latest Ably Library  -->
 
   <!-- Instance the Ably library  -->
-  <script type="text/javascript">
-    const ably = new Ably.Realtime('INSERT-YOUR-API-KEY-HERE'); /* ADD YOUR API KEY HERE */
-    channel = ably.channels.get('persistencetest'); /* Set channel to talk*/
+  <script>
+    const ably = new Ably.Realtime("ABLY_API_KEY"); /* ADD YOUR API KEY HERE */
+    channel =
+      ably.channels.get(
+        "persistencetest"
+      ); /* Channel name being published and subscribed to */
+
+    const generateRandomString = (length = 8) =>
+      Math.random().toString(20).substr(2, length);
+
+    const btn = document.getElementById("publish");
+    const results = document.getElementById("result");
 
-    ably.connection.on('connected', () => {
-      $('input[type=button]').val('Connected to Ably, click here to publish a message'); /* Confirm connection to channel */
+    btn.addEventListener("click", () => {
+      channel.publish("message", `${generateRandomString()}`);
     });
 
-    channel.subscribe('Event Name', (message) => {
-      $('#result').text(`Message from Ably: ${message.data}`); /* Change text when message received */
+    ably.connection.on("connected", () => {
+      btn.innerText = "Connected to Ably, click to publish a message";
     });
 
-    $('input[type=button]').on('click', () => {
-      channel.publish('Event Name', 'Successful publishing!'); /* Publish message to channel */
+    channel.subscribe((message) => {
+      results.innerHTML += `Message from Ably: ${message.data}<br/>`;
     });
   </script>
 </html>
 ```
 
 When you load @index.html@ and click the button a new message will be sent and be displayed on the page. You can see that connections have been made and messages sent by checking the "Stats" tab of your app's "Ably dashboard":https://ably.com/accounts.
 
-If you now navigate to your "S3 Bucket":https://s3.console.aws.amazon.com/s3/buckets and open the file we added earlier you should see it now contains the messages sent when pushing the button in "index.html". It may take a couple of minutes for AWS to process the requests.
+If you now navigate to your "S3 Bucket":https://s3.console.aws.amazon.com/s3/buckets and open the file you added earlier you should see it now contains the messages sent when pushing the button in "index.html". It may take a couple of minutes for AWS to process the requests.
```

Comment:
"Navigate to your S3 bucket and open the file you added earlier. It should now contain the ..."

(Avoids having two "now"s in one sentence)